server/src/main/java/org/apache/druid/client/BrokerServerView.java

```diff
@@ -108,25 +108,18 @@ public BrokerServerView(
     this.selectors = new HashMap<>();
     this.timelines = new HashMap<>();
 
-    this.segmentFilter = new Predicate<Pair<DruidServerMetadata, DataSegment>>()
-    {
-      @Override
-      public boolean apply(
-          Pair<DruidServerMetadata, DataSegment> input
-      )
-      {
-        if (segmentWatcherConfig.getWatchedTiers() != null
-            && !segmentWatcherConfig.getWatchedTiers().contains(input.lhs.getTier())) {
-          return false;
-        }
-
-        if (segmentWatcherConfig.getWatchedDataSources() != null
-            && !segmentWatcherConfig.getWatchedDataSources().contains(input.rhs.getDataSource())) {
-          return false;
-        }
+    this.segmentFilter = metadataAndSegment -> {
```

Comment:
Type of `metadataAndSegment` is not obvious, please use `(Pair<DruidServerMetadata, DataSegment> metadataAndSegment) -> ...`